### PR TITLE
Implemented module type of Member & done styling across the site

### PR DESCRIPTION
**Pull request by tomhuynh**

### CODE REVIEW

File: src/app/module/module-entities-add/module-entities-add.component.ts [hunk 1]
```diff
@@ -84,4 +85,13 @@ export class ModuleEntitiesAddComponent implements OnInit {
...
     field.fileDetails = value;
   }
+
+  close() {
+    this.router.navigate(['/modules/' + this.moduleId]);
+    this.moduleService.getModule(this.moduleId);
+  }
+
+  ngOnDestroy() {
+    this.moduleService.getModule(this.moduleId);
```
Comment (heathbobby) [inline src/app/module/module-entities-add/module-entities-add.component.ts]: Why do you need to do this when destroying the module component?  What would this be used for?

File: src/app/module/module-entities-add/module-entities-add.component.ts [hunk 2]
```diff
@@ -84,4 +85,13 @@ export class ModuleEntitiesAddComponent implements OnInit {
     field.fieldEntityValue.fieldValue = value.id;
     field.fileDetails = value;
   }
+
+  close() {
+    this.router.navigate(['/modules/' + this.moduleId]);
+    this.moduleService.getModule(this.moduleId);
```
Comment (heathbobby) [inline src/app/module/module-entities-add/module-entities-add.component.ts]: The module service should be state aware.  Why would you re-run this?

File: src/app/module/module.service.ts [hunk 3]
```diff
@@ -64,25 +64,33 @@ export class ModuleService {
...
-	}
-	deleteEntity(entityId: number) {
-		return this.http.delete<boolean>(`${Config.apiUrl}/ClientSite/DeleteEntityByEntityId/${entityId}`, { withCredentials: true });
-	}
-	mapEntitiesFieldForDatatable(data: any[], entityType: string, entityFieldNames: EntityTypeField[]) {
-		const datatableCols: DatatableCols[] = [];
-		this.colsValuesId = {};
-		if (entityType === 'default') {
-			data.map((value, index) => {
+  updateEntityValues(entityValue: Field[]) {
```
Comment (heathbobby) [inline src/app/module/module.service.ts]: looks like the tab alignment is off.  Can you fix this, please?

File: src/assets/scss/_variables.scss [hunk 4]
```diff
@@ -84,7 +84,7 @@ $body-color: #707070;
 $input-bg: transparent;
 $input-disabled-bg: transparent;
 $input-box-shadow: rgba(0,0,0,0);
-$input-border-color: lighten($gray-200, 1%);
+$input-border-color: #dedddd;
```
Comment (heathbobby) [inline src/assets/scss/_variables.scss]: please revert this.  I would like Dave to manage this.  If you have suggestions for style changes let me know and I will pass them along to Dave.

File: src/assets/scss/bootstrap-overrides/_forms.scss [hunk 5]
```diff
@@ -1,13 +1,12 @@
 .form-control {
-  border-left: 0;
```
Comment (heathbobby) [inline src/assets/scss/bootstrap-overrides/_forms.scss]: Let's let Dave take a look at these too.

File: src/app/module/module-entities-add/module-entities-add.component.ts (hunk 1 above)
Comment (tomhuynh) [reply src/app/module/module-entities-add/module-entities-add.component.ts]: When user is in edit entity of module page, if user clicks back arrow in browser, system should load the module site nicely for him/her. The call in ngDestroy will trigger the event bus so the module site which subscribe to that will be rendered.

File: src/app/module/module-entities-add/module-entities-add.component.ts (hunk 2 above)
Comment (tomhuynh) [reply src/app/module/module-entities-add/module-entities-add.component.ts]: Removed line 91 b/c the call in ngDestroy is taking care of this.

File: src/assets/scss/bootstrap-overrides/_forms.scss (hunk 5 above)
Comment (tomhuynh) [reply src/assets/scss/bootstrap-overrides/_forms.scss]: Reverted.

**Pull request merged**